src/Drivers/Parsian/Parsian.php

```diff
@@ -0,0 +1,467 @@
+            '59' => 'ﻛﺎﺭﺕ ﻣﻈﻨﻮﻥ ﺑﻪ ﺗﻘﻠﺐ ﺍﺳﺖ 	',
+            '60' => 'ﺑﻨﺎﺑﺮ ﺗﻮﺻﻴﻪ ﻣﻮﺳﺴﻪ ﻳﺎ ﺑﺎﻧﻚ ﻣﺪﻳﺮ ﻛﺎﺭﺕ ﺑﻪ ﭘﺬﻳﺮﻧﺪﻩ ﻛﺎﺭﺕ ، ﺗﺮﺍﻛﻨﺶ ﺩﺭﺧﻮﺍﺳﺘﻲ ﭘﺬﻳﺮﻓﺘﻪ ﻧﻤﻲ ﺷﻮﺩ ',
+            '61' => 'ﻣﺒﻠﻎ ﺗﺮﺍﻛﻨﺶ ﻛﻤﺘﺮ ﺍﺯ ﺣﺪ ﺗﻌﻴﻴﻦ ﺷﺪﻩ ﺗﻮﺳﻂ ﺻﺎﺩﺭ ﻛﻨﻨﺪﻩ ﻛﺎﺭﺕ ﻭ ﻳﺎ ﺑﻴﺸﺘﺮ ﺍﺯ ﺣﺪ ﻣﺠﺎﺯ ﻣﻲ ﺑﺎﺷﺪ 	',
+            '62' => 'ﻛﺎﺭﺕ ﻣﺤﺪﻭﺩ ﺷﺪﻩ ﺍﺳﺖ 	',
+            '63' => 'ﺗﻤﻬﻴﺪﺍﺕ ﺍﻣﻨﻴﺘﻲ ﻧﻘﺾ ﮔﺮﺩﻳﺪﻩ ﺍﺳﺖ 	',
+            '64' => 'ﻣﺒﻠﻎ ﺗﺮﺍﻛﻨﺶ ﺍﺻﻠﻲ ﻥ ﺍﻣﻌﺘﺒﺮ ﺍﺳﺖ. (ﺗﺮﺍﻛﻨﺶ ﻣﺎﻟﻲ ﺍﺻﻠﻲ ﺑﺎ ﺍﻳﻦ ﻣﺒﻠﻎ ﻧﻤﻲ ﺑﺎﺷﺪ) 	',
+            '65' => 'ﺗﻌﺪﺍﺩ ﺩﺭﺧﻮﺍﺳﺖ ﺗﺮﺍﻛﻨﺶ ﺑﻴﺶ ﺍﺯ ﺣﺪ ﻣﺠﺎﺯ ﻣﻲ ﺑﺎﺷﺪ 	',
+            '66' => 'ﺩﺭ ﭘﻲ ﺗﺮﺍﻛﻨﺶ ﺩﺭﺧﻮﺍﺳﺘﻲ ﭘﺬﻳﺮﻧﺪﻩ ﺑﺎ ﺑﺨﺶ ﺍﻣﻨﻴﺘﻲ ﻣﻮﺳﺴﻪ ﻳﺎ ﺑﺎﻧﻚ ﺗﻤﺎﺱ ﮔﺮﻓﺘﻪ ﺍﺳﺖ ( ﻭ ﻳﺎ ﻣﻴﮕﻴﺮﺩ ) 	',
+            '67' => 'ﻛﺎﺭﺕ ﺗﻮﺳﻂ ﺩﺳﺘﮕﺎﻩ ﺿﺒﻂ ﺷﻮﺩ 	',
+            '68' => 'ﭘﺎﺳﺦ ﻻﺯﻡ ﺑﺮﺍﻱ ﺗﻜﻤﻴﻞ ﻳﺎ ﺍﻧﺠﺎﻡ ﺗﺮﺍﻛﻨﺶ ﺧﻴﻠﻲ ﺩﻳﺮ ﺭﺳﻴﺪﻩ ﺍﺳﺖ 	',
+            '69' => 'ﺗﻌﺪﺍﺩ ﺩﻓﻌﺎﺕ ﺗﻜﺮﺍﺭﺭﻣﺰ ﺍﺯ ﺣﺪ ﻣﺠﺎﺯ ﮔﺬﺷﺘﻪ ﺍﺳﺖ 	',
+            '75' => 'ﺗﻌﺪﺍﺩ ﺩﻓﻌﺎﺕ ﻭﺭﻭﺩ ﺭﻣﺰﻏﻠﻂ ﺑﻴﺶ ﺍﺯ ﺣﺪﻣﺠﺎﺯ ﺍﺳﺖ 	',
+            '76' => 'ﻣﺒﻠﻎ ﺍﻧﺘﻘﺎﻝ ﺩﺍﺩﻩ ﺷﺪﻩ ﻣﻌﺘﺒﺮ ﻧﻴﺴﺖ 	',
+            '77' => 'ﺭﻭﺯ ﻣﺎﻟﻲ ﺗﺮﺍﻛﻨﺶ ﻧﺎ ﻣﻌﺘﺒﺮ ﺍﺳﺖ ﻳﺎ ﻣﻬﻠﺖ ﺯﻣﺎﻥ ﺍﺭﺳﺎﻝ ﺍﺻﻼﺣﻴﻪ ﺑﻪ ﭘﺎﻳﺎﻥ ﺭﺳﻴﺪﻩ ﺍﺳﺖ 	',
+            '78' => 'ﻛﺎﺭﺕ ﻓﻌﺎﻝ ﻧﻴﺴﺖ 	',
+            '79' => 'ﺣﺴﺎﺏ ﻣﺘﺼﻞ ﺑﻪ ﻛﺎﺭﺕ ﻧﺎ ﻣﻌﺘﺒﺮ ﺍﺳﺖ ﻳﺎ ﺩﺍﺭﺍﻱ ﺍﺷﻜﺎﻝ ﺍﺳﺖ 	',
+            '80' => 'ﺩﺭﺧﻮﺍﺳﺖ ﺗﺮﺍﻛﻨﺶ ﺭﺩ ﺷﺪﻩ ﺍﺳﺖ 	',
+            '81' => 'ﻛﺎﺭﺕ ﭘﺬﻳﺮﻓﺘﻪ ﻧﺸﺪ ( ﺍﺧﺘﺼﺎﺻﻲ ( SLM 	',
+            '82' => 'ﭘﻴﺎﻡ ﺗﺎﻳﻴﺪ ﺍﺯ ﺩﺳﺘﮕﺎﻩ ﺧﻮﺩ ﭘﺮﺩﺍﺯ ﺩﺭﻳﺎﻓﺖ ﻧﺸﺪﻩ ﺍﺳﺖ ( ﺍﺧﺘﺼﺎﺻﻲ ( SLm 	',
+            '83' => 'ﺳﺮﻭﻳﺲ ﮔﺮ ﺳﻮﺋﻴﭻ ﻛﺎﺭﺕ ﺗﺮﺍﻛﻨﺶ ﺭﺍ ﻧﭙﺬﻳﺮﻓﺘﻪ ﺍﺳﺖ 	',
+            '84' => 'ﺩﺭ ﺗﺮﺍﻛﻨﺸﻬﺎﻳﻲ ﻛﻪ ﺍﻧﺠﺎﻡ ﺁﻥ ﻣﺴﺘﻠﺰﻡ ﺍﺭﺗﺒﺎﻁ ﺑﺎ ﺻﺎﺩﺭ ﻛﻨﻨﺪﻩ ﺍﺳﺖ ﺩﺭ ﺻﻮﺭﺕ ﻓﻌﺎﻝ ﻧﺒﻮﺩﻥ ﺻﺎﺩﺭ ﻛﻨﻨﺪﻩ ﺍﻳﻦ ﭘﻴﺎﻡ ﺩﺭ ﭘﺎﺳﺦ ﺍﺭﺳﺎﻝ ﺧﻮﺍﻫﺪ ﺷﺪ 	',
+            '85' => 'ﭘﺮﺩﺍﺯﺵ ﮔﺮ ﻭ ﻳﺎ ﻣﺒﺪﺍ ﺍﻧﺠﺎﻡ ﺗﺮﺍﻛﻨﺶ ﻣﻌﺘﺒﺮ ﻧﻴﺴﺖ 	',
+            '86' => 'ﺗﺮﺍﻛﻨﺶ ﺩﺭﺧﻮﺍﺳﺘﻲ ﺑﺮﺍﻱ ﺑﺨﺶ ﺳﺨﺖ ﺍﻓﺰﺍﺭﻱ ﺩﺭﺧﻮﺍﺳﺖ ﺷﺪﻩ ﺍﺯ ﺁﻥ ﻗﺎﺑﻞ ﻗﺒﻮﻝ ﻧﻴﺴﺖ 	',
+            '87' => 'ﺳﻴﺴﺘﻢ ﺩﺭ ﺗﺒﺎﺩﻝ ﻛﻠﻴﺪ ﺭﻣﺰ ﺩﭼﺎﺭ ﻣﺸﻜﻞ ﺷﺪﻩ ﺍﺳﺖ ( ﻛﺪ ﭘﺎﺳﺦ ﺍﺧﺘﺼﺎﺻﻲ ( SLM 	',
+            '88' => 'ﺳﻴﺴﺘﻢ ﺩﺭ ﺗﺒﺎﺩﻝ ﻛﻠﻴﺪ MAC ﺩﭼﺎﺭ ﻣﺸﻜﻞ ﺷﺪﻩ ﺍﺳﺖ (ﻛﺪ ﭘﺎﺳﺦ ﺍﺧﺘﺼﺎﺻﻲ ( SLM 	',
+            '89' => 'ﻋﺪﻡ ﺗﺎﻳﻴﺪ ﺗﺮﺍﻛﻨﺶ ﺗﻮﺳﻂ ﺳﻮﺋﻴﭻ ﺧﺎﺭﺟﻲ ( ﻛﺪ ﭘﺎﺳﺦ ﺍﺧﺘﺼﺎﺻﻲ ( SLM 	',
+            '90' => 'ﺳﺎﻣﺎﻧﻪ ﻣﻘﺼﺪ ﺗﺮﺍﻛﻨﺶ ﺩﺭﺣﺎﻝ ﺍﻧﺠﺎﻡ ﻋﻤﻠﻴﺎﺕ ﭘﺎﻳﺎﻥ ﺭﻭﺯ ﻣﻲ ﺑﺎﺷﺪ 	',
+            '91' => 'ﺳﻴﺴﺘﻢ ﺻﺪﻭﺭ ﻣﺠﻮﺯ ﺍﻧﺠﺎﻡ ﺗﺮﺍﻛﻨﺶ ﻣﻮﻗﺘﺎ ﻏﻴﺮ ﻓﻌﺎﻝ ﺍﺳﺖ ﻭ ﻳﺎ ﺯﻣﺎﻥ ﺗﻌﻴﻴﻦ ﺷﺪﻩ ﺑﺮﺍﻱ ﺻﺪﻭ ﻣﺠﻮﺯ ﺑﻪ ﭘﺎﻳﺎﻥ ﺭﺳﻴﺪﻩ ﺍﺳﺖ 	',
+            '92' => 'ﻣﻘﺼﺪ ﺗﺮﺍﻛﻨﺶ ﭘﻴﺪﺍ ﻧﺸﺪ 	',
+            '93' => 'ﺍﻣﻜﺎﻥ ﺗﻜﻤﻴﻞ ﺗﺮﺍﻛﻨﺶ ﻭﺟﻮﺩ ﻧﺪﺍﺭﺩ 	',
+            '94' => 'ﺍﺭﺳﺎﻝ ﺗﻜﺮﺍﺭﻱ ﺗﺮﺍﻛﻨﺶ ﺑﻮﺟﻮﺩ ﺁﻣﺪﻩ ﺍﺳﺖ 	',
+            '95' => 'ﺩﺭ ﻋﻤﻠﻴﺎﺕ ﻣﻐﺎﻳﺮﺕ ﮔﻴﺮﻱ ﺗﺮﻣﻴﻨﺎﻝ ﺍﺷﻜﺎﻝ ﺭﺥ ﺩﺍﺩﻩ ﺍﺳﺖ 	',
+            '96' => 'ﺍﺷﻜﺎﻝ ﺩﺭ ﻋﻤﻠﻜﺮﺩ ﺳﻴﺴﺘﻢ 	',
+            '97' => 'ﺗﺮﺍﻛﻨﺶ ﺍﺯ ﺳﻮﻱ ﺻﺎﺩﺭﻛﻨﻨﺪﻩ ﻛﺎﺭﺕ ﺭﺩ ﺷﺪﻩ ﺍﺳﺖ 	',
+            '99' => 'ﺧﻄﺎﻱ ﺻﺎﺩﺭ ﻛﻨﻨﺪﮔﻲ 	',
+            '200' => 'ﺳﺎﻳﺮ ﺧﻄﺎﻫﺎﻱ ﻧﮕﺎﺷﺖ ﻧﺸﺪﻩ ﺳﺎﻣﺎﻧﻪ ﻫﺎﻱ ﺑﺎﻧﻜﻲ '
+
+        ];
+
+        return array_key_exists($statusCode, $messages) ? $messages[$statusCode] : $statusCode;
+    }
+
+    protected function getSuccessResponseStatusCode(): int
+    {
+        return  0;
+    }
+
+    protected function getPurchaseUrl(): string
```

Comment:
this is not used

Reply:
this method is abstract and it's not possible to remove